Rakefile

```diff
@@ -73,3 +73,25 @@ module Releaser
     File.write(path, yield(content))
   end
 end
+
+desc "Check for stale <<next>> placeholders when VERSION is updated"
+task :check_version_placeholders do
+  # Check if VERSION file was modified in the last commit
+  version_changed = %x(git diff HEAD~1 HEAD --name-only).split("\n").include?("VERSION")
```

Comment:
Should we check `lib/rubocop/sorbet/version.rb` too in case someone bumps it directly?

Reply:
I actually like having the `VERSION` constant exist, so I'll suggest that we read the content from the file. What do you think?

Reply:
That works too. Updated.